aea/cli/add.py

```diff
@@ -186,31 +184,38 @@ def _add_item_deps(
                 add_item(ctx, SKILL, skill_public_id)
 
 
-def find_item_locally_or_distributed(
-    ctx: Context, item_type: str, item_public_id: PublicId, dest_path: str
+def fetch_item_remote(item_type: str, public_id: PublicId, cwd: str, dest: str) -> Path:
+    """Fetch item from a rmeote backend."""
+
+    registry_config = get_registry_config()
+    registry_type = registry_config.get("default")
+
+    click.echo(f"Using registry: {registry_type} ")
+    if registry_type == REGISTRY_HTTP:
+        return fetch_package(item_type, public_id=public_id, cwd=cwd, dest=dest)
+    else:  # pylint: disable=no-else-return
+        return fetch_ipfs(item_type, public_id=public_id, cwd=cwd, dest=dest)
+
+
+def fetch_item_locally(
+    ctx: Context, item_type: str, public_id: PublicId, dest_path: str
 ) -> Path:
     """
     Unify find item locally both in case it is distributed or not.
 
     :param ctx: the CLI context.
     :param item_type: the item type.
-    :param item_public_id: the item public id.
+    :param public_id: the item public id.
     :param dest_path: the path to the destination.
     :return: the path to the found package.
     """
-    is_distributed = is_distributed_item(item_public_id)
-    if is_distributed:  # pragma: nocover
-        source_path = find_item_in_distribution(ctx, item_type, item_public_id)
-        package_path = copy_package_directory(source_path, dest_path)
-    else:
-        source_path, _ = find_item_locally(ctx, item_type, item_public_id)
-        package_path = copy_package_directory(source_path, dest_path)
```

Comment:
why has this been changed?

Reply:
This is unused logic, I analyzed the code for finding distributed items and traced it back to the variable containing public ids for distributed items (`DISTRIBUTED_PACKAGES: List[str] = []`) it's empty and there's no additional logic to update the list. Should I add the logic back again?

Reply:
Yes please add it because if we do re-introduce distributed packages then we already have the logic. All the other logic is still in place for it too.